services/worker/README.md

```diff
@@ -92,7 +92,7 @@ Set environment variables to configure the `duckdb-index` worker (`DUCKDB_INDEX_
 - `DUCKDB_INDEX_CACHE_DIRECTORY`: directory where the temporal duckdb index files are stored. Defaults to empty.
 - `DUCKDB_INDEX_COMMIT_MESSAGE`: the git commit message when the worker uploads the duckdb index file to the Hub. Defaults to `Update duckdb index file`.
 - `DUCKDB_INDEX_COMMITTER_HF_TOKEN`: the HuggingFace token to commit the duckdb index file to the Hub. The token must be an app token associated with a user that has the right to 1. create the `refs/convert/parquet` branch (see `DUCKDB_INDEX_TARGET_REVISION`) and 2. push commits to it on any dataset. [Datasets maintainers](https://huggingface.co/datasets-maintainers) members have these rights. The token must have permission to write. If not set, the worker will fail. Defaults to None.
-- `DUCKDB_INDEX_MAX_DATASET_SIZE_BYTES`: the maximum size in bytes of the dataset's parquet files to index. Datasets with bigger size are ignored. Defaults to `100_000_000`.
+- `DUCKDB_INDEX_MAX_DATASET_SIZE_BYTES`: size in bytes by which the number of first `n` parquet files to index is determined, satisfies the following condition: `sum(parquet_files[0] + ... + parquet_files[n-1]) <= DUCKDB_INDEX_MAX_DATASET_SIZE_BYTES & sum(parquet_files[0] + ... + parquet_files[n]) > DUCKDB_INDEX_MAX_DATASET_SIZE_BYTES`. Defaults to `100_000_000`.
```

Comment:
idk how to explain this more clearly....... but this is exactly what we do (from my understanding) and not what is said everywhere (that this number is the exact maximum)

Reply:
is it the sum of the parquet files? in which case, it could contain a lot more than 5GB of uncompressed "data".